cupydo/interfaces/Pfem.py

```diff
@@ -179,7 +180,8 @@ def applyNodalDisplacements(self, dx, dy, dz, dx_nM1, dy_nM1, dz_nM1, haloNodesD
     def update(self, dt):
         self.pfem.scheme.t+=dt
         self.pfem.scheme.nt+=1
-        self.pfem.scheme.updateSolutionVectors(self.V,self.V0,self.u,self.v,self.p,self.velocity)
+        self.pfem.scheme.updateSolutionVectors(self.V,self.V0,self.u,self.v,self.p,self.velocity),self.matID
```

Comment:
Is this right? It looks a bit weird

Reply:
oh damn that is a copy paste mistake. I will fix it